src/main/java/org/bigredbands/mb/utils/PDFStringUtils.java

```diff
@@ -11,4 +11,49 @@ public static float stringWidth(String str, PDFont font,
     {
         return font.getStringWidth(str) / 1000.0f * fontSize;
     }
+
+    public static String[] wordWrap(String stringToWrap,
+        float widthLimit, float initialOffset,
+        String wrapOn,
+        PDFont font, float fontSize) throws IOException
+    {
+        ArrayList<String> strs = new ArrayList<String>();
+        String[] tokens = stringToWrap.split(wrapOn);
+
+        float remainingLine = widthLimit - initialOffset;
+        String line = "";
+        for (int i = 0; i < tokens.length; i++) {
+            String token = tokens[i];
+            if (i < tokens.length - 1) token += wrapOn;
+
+            float tokenSize = stringWidth(token, font, fontSize);
+
+            // The token fits on the current line
+            if (remainingLine >= tokenSize) {
+                line += token;
+                remainingLine -= tokenSize;
+                continue;
+            }
+
+            // Token doesn't fit on the line
+            if (line.isEmpty()) {
+                // Just put the token on its own line (even though it's too long)
+                strs.add(token.trim());
```

Comment:
There's got to be way to split the token across two lines if it's too long to fit on one.

Reply:
There probably is, I suppose someone could write a token that's way too long. I've added an issue to address this (FWIW, I never ran into this case on showcomm as far as I remember): https://github.com/bigredbands/rank-panda/issues/28